### PR TITLE
Adds fb token negotiation and adds (temporary) code for posting exter…

### LINKED ISSUE #455: Negotiate token authorization with Faceboook

### PR DESCRIPTION
**Pull request by fellmirr**
…nal donations to fb

Closes #455 

For some reason all of the schema types were updated when generating the schema, not sure why, does not seem to be important

### CODE REVIEW

File: src/routes/facebook.ts [hunk 1]
```diff
@@ -314,4 +315,20 @@ router.post(
   }
 );
 
+// Route to get token
```
Comment (fellmirr) [inline src/routes/facebook.ts:318]: This needs to go!

File: src/custom_modules/facebook.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { Donation, Fundraiser } from "../schemas/types";
+import { createHash } from "crypto";
+import luxon from "luxon";
+import fetch from "node-fetch";
+import config from "../config.js";
+
+export const fetchToken = async () => {
+  console.log(config.facebook_sync_app_id, config.facebook_sync_app_secret);
```
Comment (EliseAndresen) [inline src/custom_modules/facebook.ts:8]: console.log

**Review by EliseAndresen — CHANGES_REQUESTED** [review]
Good to go når kode snutten i src/routes/facebook.ts er slettet

**Review by EliseAndresen — APPROVED** [review]

File: src/custom_modules/facebook.ts (hunk 2 above)
Comment (fellmirr) [reply src/custom_modules/facebook.ts]: Done

**Pull request merged**